concrete/src/Entity/User/LoginAttempt.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Concrete\Core\Entity\User;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="Concrete\Core\Entity\User\LoginAttemptRepository")
+ * @ORM\Table(
+ *     name="UserLoginAttempts"
+ * )
+ */
+class LoginAttempt
```

Comment:
Shouldn't we have a migration? Otherwise the `UserLoginAttempts` database table won't be created.

Reply:
Is `concrete/src/Updater/Migrations/Migrations/Version20181029223809.php` not enough?

Reply:
:+1: